### DISCUSSION TITLE
Android client

### DISCUSSION THREAD

**Discussion by Nortonko** [narrowlink/narrowlink#107] [Q&A] [answered]
Hello,
thank you for your excellent work. I have a question. Is it possible to connect directly from android phone as a client to my Narrowlink network?

**Comment by SajjadPourali** [comment, marked as answer]
Hello,

Thanks. Technically, it is possible to run a Narrowlink client in a terminal environment like Termux and run the proxy. However, I don't recommend it as it requires the orchestration of services and can be complicated.

There is a future plan to create an Android app for this purpose. https://github.com/narrowlink/narrowlink/issues/108
